include/dem/volume_insertion.h

```diff
@@ -12,13 +12,12 @@
  * the top level of the Lethe distribution.
  *
  * ---------------------------------------------------------------------
-
  *
- * Author: Shahab Golshan, Polytechnique Montreal, 2019
  */
 
 #include <dem/dem_solver_parameters.h>
 #include <dem/insertion.h>
+#include <dem/normal_distribution.h>
```

Comment:
Is this needed? You are not using anything from the normal distribution in this class, are you?